docs/index.md

```diff
@@ -5,7 +5,7 @@ layout: home
 hero:
   name: "micromissiles-unity"
   
-  tagline: "Swarm-on-swarm simulator using micromissiles for point defense"
+  tagline: "A large-scale air defense simulation platform for simulating swarm-on-swarm engagements with hierarchical interceptors and micromissiles"
   actions:
     - theme: brand
       text: Documentation
```

Comment:
"Documentation" here is so general/vague doesn't really help the reader know what they're about to click on lol. Suggest changing this to Simulator Overview in both index.md and config.mts